### PR TITLE
docs: use make cmd instead of docker build cmd

### PR DESCRIPTION
**Pull request by gxthrj**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [ ] Bugfix
- [ ] New feature provided
- [ ] Improve performance
- [ ] Backport patches

- Related issues

___
### Bugfix
- Description

- How to fix?

___
### New feature or improvement
- Describe the details and related test reports.

___
### Backport patches
- Why need to backport?

- Source branch

- Related commits and pull requests

- Target branch

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ helm install ingress-apisix -n ingress-apisix ./charts/ingress-apisix
 ## Master branch builds
 
 ```shell
-docker build -t apache/ingress-controller:v0.1.0 ../.
+cd .. && make build-image
```
Comment (membphis) [inline docs/install.md:74]: why do we need `cd .. &&` here?

Comment (tokers) [reply docs/install.md:74]: @gxthrj

Comment (gxthrj) [reply docs/install.md:74]: The file `install.md` is in the `docs` path, but `Makefile` is in the root directory.

Comment (tokers) [reply docs/install.md:74]: We don't need to edit commands from the perspective of the path of `install.md`.

```shell
$ cd /path/to/apisix-ingress-controller
$ make build-image
```

I think that's enough.

Comment (gxthrj) [reply docs/install.md:74]: OK

**Review by tokers — APPROVED** [review]

**Comment by tokers** [comment]
@membphis Please check it out.

**Review by membphis — APPROVED** [review]

**Pull request merged**